src/main/java/com/datalogics/pdf/samples/forms/FillForm.java

```diff
@@ -177,8 +177,6 @@ public static void fillAcroformFdf(final PDFDocument pdfDocument, final URL inpu
         final FDFService fdfService = new FDFService(pdfDocument);
         fdfService.importForm(fdfDocument);
 
-        // Run calculations on the AcroForm...
-        FormFieldService.getAcroFormFieldManager(pdfDocument).runCalculateScripts();
```

Comment:
@datalogics-kam I understand what went wrong, and I maintain that it is not a problem with the ``XFDFService`` code but with the PDF we're using the ``FillForm`` sample. To verify what you're trying to prove, we'll need to make substantive changes to the sample and the test.

If both @datalogics-kam and @datalogics-coreyy this will be easier for me to demonstrate in person tomorrow.